package.json

```diff
@@ -56,7 +57,7 @@
     "cross-env": "5.1.2",
     "file-saver": "^1.3.3",
     "react": "16.2.0",
```

Comment:
Any reason we aren't bumping `react` as well?

Reply:
The react-dom version was updated as my editor was coming up with a message saying that it had a security vulnerability. I did try to update React too, but did come across some build issues that felt beyond the scope of this work. I think it would be worthwhile having a tech debt ticket covering this work separately